src/pages/LiveChat/components/messages/Message.tsx

```diff
@@ -2,31 +2,31 @@ import { CollectionReference, deleteDoc, doc, DocumentData } from "firebase/fire
 import { MutableRefObject } from "react";
 import { useOutletContext } from "react-router-dom";
 import { MessageType } from "../LiveChat";
+import { MessageActions } from "./MessagesList";
 
 type Props = {
   message: MessageType,
   messagesListDb: CollectionReference<DocumentData>,
-  messagesListActionRef: MutableRefObject<{ action: string; }>
```

Comment:
Updated action from string to a custom Enum, don't know why I didn't do it like this from the start.